projects/components/src/panel/panel.component.test.ts

```diff
@@ -56,9 +59,11 @@ describe('Panel component', () => {
     expect(spectator.query('.title')).toHaveText('Results');
     expect(spectator.query('.summary')).toHaveText('(508 Records)');
     expect(spectator.query('.body')).not.toExist();
+    expect(spectator.query('.collapsed-body')).toHaveText('Collapsed Body Content');
 
     spectator.click('.title');
     expect(spectator.query('.body')).toHaveText('Body Content');
+    expect(spectator.query('.collapsed-body')).not.toExist();
```

Comment:
also add a test to check absence of collapsed-body when its not supplied. i.e, the default case.

Reply:
done